File: Project_Files/social_club/Start_Python/member.py
from activity import Activity


class Member:
    def __init__(self, new_id, new_first_name, new_last_name, new_birth_date):
        self.id = new_id
        self.first_name = new_first_name
        self.last_name = new_last_name
        self.birth_date = new_birth_date
        self.all_my_activities = []

    def sort_activities(self):
        self.all_my_activities = sorted(self.all_my_activities, key=lambda activity: activity.start_date)

File: Project_Files/Tests/JavaScript_SocialClub_Python_Bloodbank/test/Start_JavaScript/src/Member.js
class Member {
  constructor (newId, newFirstName, newLastName, newBirthDate) {
    this.id = newId
    this.firstName = newFirstName
    this.lastName = newLastName
    this.birthDate = newBirthDate
    this.allMyActivities = []
  }
  sortActivities () {
    this.allMyActivities.sort(function (a, b) {
      return a.startDate - b.startDate
    })
  }
}
